<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DATA_LIST</class>
 <widget class="QDialog" name="DATA_LIST">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data List</string>
  </property>
  <property name="statusTip">
   <string notr="true">Item Count </string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>561</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Filters</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Inst Family</string>
    </property>
   </widget>
   <widget class="QComboBox" name="DataList_Family_select">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Any</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Keyboards</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Strings</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Winds</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Brass</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Percussion</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ChromPerc</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Voice</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Synth</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Guitars</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bass</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ensemble</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ethnic</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SFX</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="DataList_Name_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>full / partial name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DataList_Select_button">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DataList_Exit_button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QComboBox" name="DataList_Type_button">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Performance</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Patch</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tone/Wave</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rhythm</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>52</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="DataList_Result_list">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>561</width>
     <height>271</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::NoWrap</enum>
   </property>
   <property name="tabStopWidth">
    <number>4</number>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
